internal/command/machine/clone.go

```diff
@@ -251,6 +251,9 @@ func runMachineClone(ctx context.Context) (err error) {
 		}
 		targetConfig.Standbys = lo.Ternary(len(standbys) > 0, standbys, nil)
 		targetConfig.Env["FLY_STANDBY_FOR"] = strings.Join(standbys, ",")
+		targetConfig.Env = lo.Assign(targetConfig.Env,
+			map[string]string{"FLY_STANDBY_FOR": strings.Join(standbys, ",")},
+		)
```

Comment:
do we need to remove the original line here?

```suggestion
		targetConfig.Env = lo.Assign(targetConfig.Env,
			map[string]string{"FLY_STANDBY_FOR": strings.Join(standbys, ",")},
		)
```

Reply:
oops, yes

Reply:
I'm also not sure if `lo.Assign` works with a potentially-`nil` first argument, though the existing test should check this